include/multipass/ip_address.h

```diff
@@ -40,9 +40,10 @@ struct IPAddress
     bool operator<=(const IPAddress& other) const;
     bool operator>(const IPAddress& other) const;
     bool operator>=(const IPAddress& other) const;
+    IPAddress operator=(const IPAddress& other);
     IPAddress operator+(int value) const;
 
-    const std::array<uint8_t, 4> octets;
+    std::array<uint8_t, 4> octets;
```

Comment:
There should be no need to remove const here.

IPAddress was made so that it's invariant after construction.